setup.py

```diff
@@ -10,8 +10,8 @@
     license="MIT",
     url="https://github.com/sinzlab/mei",
     keywords="feature visualization MEI pytorch",
+    python_requires="<3.10",
     packages=["mei"],
```

Comment:
from pip.req import parse_requirements
from pip.download import PipSession

install_reqs = parse_requirements(<requirements_path>, session=PipSession())

reqs = [str(ir.req) for ir in install_reqs]

setup(
    ...
    install_requires=reqs
    ....
)